cmd/up/ctx/cmd.go

```diff
@@ -41,7 +41,7 @@ import (
 )
 
 const (
-	contextSwitchedFmt = "Kubeconfig context %q switched to: %s\n"
+	contextSwitchedFmt = "Added new context to kubeconfig: %s\n"
```

Comment:
🤔  we are doing more than added a context, we are changing the current context. 

what about:
```suggestion
	contextSwitchedFmt = "Switched kubeconfig context to %q\n"
```

Reply:
Updated